webroot/js/app.js

```diff
@@ -383,9 +383,45 @@ export default class App extends Component {
     }
   }
 
+  handleMuteKeyPressed() {
+    this.player.vjsPlayer.muted(!this.player.vjsPlayer.muted());
+  }
+
+  handleFullScreenKeyPressed() {
+    if (this.player.vjsPlayer.isFullscreen()) {
+      this.player.vjsPlayer.exitFullscreen();
+    } else {
+      this.player.vjsPlayer.requestFullscreen();
+    }
+  }
+
+  handleVolumeSet(factor) {
+    this.player.vjsPlayer.volume(this.player.vjsPlayer.volume() + factor);
+  }
+
   handleKeyPressed(e) {
-    if (e.code === 'Space' && e.target === document.body && this.state.streamOnline) {
-      this.handleSpaceBarPressed(e);
+    if (e.target === document.body && this.state.streamOnline) {
```

Comment:
So I noticed that if you initially click to play from the play button, the focus stays on the player, and then the keyboard commands won't work. However, they will work if you take out the e.target check. I don't quite recall why the check is there. Can we remove it?

```suggestion
    if (this.state.streamOnline) {
```

Reply:
This check exists to ensure that keyboard controls are not tiggered as long as a chat input is focussed. Otherwise users would constantly start and stop their video when typing chat messages.